ApollosProject/apollos-prototype
# [OnBoarding] Adds Connected AboutYou Component

## DESCRIPTION

As a User, I want my personal details to be as up to date as possible so my experience can be as personalized as possible.

### What does this PR do, or why is it needed?

This is needed to communicate person changes to Rock.

### What design trade-offs/decisions were made?

N/A

### How do I test this PR?

You can launch the Onboarding Slides and after entering your gender and birthdate, you hit the Next button. You can then navigate to your rock profile to see if the changes took place.

### What automated tests did you write?

Snapshots for `AboutYouConnected`

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
